lib/compiler/src/compile.erl

```diff
@@ -130,27 +131,9 @@ See `m:erl_id_trans` for an example and an explanation of the function
 
 [](){: #error_information }
```

Comment:
You probably want to move/remove this anchor.

Reply:
Thanks! Done.